fri/src/prover.rs

```diff
@@ -109,24 +110,25 @@ fn answer_query<FC: FriConfig>(
 fn commit_phase<FC: FriConfig>(
     config: &FC,
     input_mmcs: &[FC::InputMmcs],
-    input_data: &[&<FC::InputMmcs as Mmcs<FC::Challenge>>::ProverData],
+    input_data: &[&<FC::InputMmcs as Mmcs<FC::Val>>::ProverData],
     log_max_height: usize,
     challenger: &mut FC::Challenger,
 ) -> CommitPhaseResult<FC> {
     let max_height = 1 << log_max_height;
-    let matrices_with_log_height = |log_height| {
```

Comment:
shoutout to @andrewtian for noticing this takes around 7ms per call (it builds the denominators)